src/render_output_ppm.c

```diff
@@ -37,21 +34,47 @@ static void	ft_write_ppm_header(int width, int height, int fd)
 	ft_putchar_fd('\n', fd);
 }
 
-t_err	ft_output_as_ppm(int *img_arr, int width, int height)
+static void	ft_write_to_file(int fd, int *pix_arr, int pixel_sum)
 {
-	int	fd;
-	int	pixel_sum;
 	int	i;
 
-	fd = 0;
-	if (ft_err_open("outfile.ppm", O_CREAT | O_WRONLY | O_TRUNC, &fd))
-		return (ERROR);
-	ft_write_ppm_header(width, height, fd);
-	pixel_sum = width * height;
 	i = 0;
 	while (i < pixel_sum)
-		ft_write_pixel(img_arr[i++], fd);
+	{
+		ft_write_pixel(pix_arr[i++], fd);
+		if (i % 100 == 0)
```

Comment:
What's that condition do?

Reply:
While printing we output a percentage on the terminal. I added it to indicate how much longer you have to wait. Printing the image is surprisingly slow! But yeah it needs to write every single pixel to the file. 

This condition says, that we only want to output the current % for every 100 pixel. 
I think printing % for every pixel would slow it down further.